test/pkg/ptphelper/ptphelper.go

```diff
@@ -598,6 +606,34 @@ func GetPtpOperatorVersion() (string, error) {
 	return ptpOperatorVersion, err
 }
 
+func GetPtpOperatorVersionFromImage() (string, error) {
```

Comment:
Might be better to get it from the BUILD_VERSION or RELEASE_VERSION in the env of the container.